cluster/whitelist.go

```diff
@@ -0,0 +1,58 @@
+package cluster
+
+import (
+	"github.com/banzaicloud/pipeline/helm"
+	"github.com/banzaicloud/pipeline/pkg/security"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/rest"
+)
+
+const GroupName = "security.banzaicloud.com"
+const GroupVersion = "v1alpha1"
+
+var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: GroupVersion}
+
+var (
+	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
+	AddToScheme   = SchemeBuilder.AddToScheme
+)
+
+func addKnownTypes(scheme *runtime.Scheme) error {
+	scheme.AddKnownTypes(SchemeGroupVersion,
+		&security.WhiteListItem{},
+		&security.WhiteList{},
+	)
+
+	metav1.AddToGroupVersion(scheme, SchemeGroupVersion)
+	return nil
+}
+
+func GetWhitelists(cc CommonCluster) (*security.WhiteList, error) {
+	result := &security.WhiteList{}
+	// Get kubernetes configuration
+	kubeConfig, err := cc.GetK8sConfig()
+	if err != nil {
+		return nil, err
+	}
+	// Get customresource Client
+	clientCfg, err := helm.GetK8sClientConfig(kubeConfig)
```

Comment:
The code for instantiating a client for manipulating CRD could be used in other places as well, thus I think it worth moving it to a common place.